analysis/magefrost/src/normalizers/CometStormLinkNormalizer.ts

```diff
@@ -0,0 +1,45 @@
+import SPELLS from 'common/SPELLS';
+import EventLinkNormalizer, { EventLink } from 'parser/core/EventLinkNormalizer';
+import {
+  CastEvent,
+  DamageEvent,
+  EventType,
+  GetRelatedEvents,
+  HasRelatedEvent,
+} from 'parser/core/Events';
+import { Options } from 'parser/core/Module';
+
+const CAST_BUFFER_MS = 2500;
+
+export const FROM_HARDCAST = 'FromHardcast';
+export const HITS_TARGET = 'HitsTarget';
+
+const EVENT_LINKS: EventLink[] = [
+  {
+    linkRelation: FROM_HARDCAST,
+    linkingEventId: SPELLS.COMET_STORM_DAMAGE.id,
+    linkingEventType: EventType.Damage,
+    referencedEventId: SPELLS.COMET_STORM_TALENT.id,
+    referencedEventType: EventType.Cast,
+    forwardBufferMs: CAST_BUFFER_MS,
+    backwardBufferMs: CAST_BUFFER_MS,
```

Comment:
2500 ms for the back buffer is almost certainly too large - you should do some logs analysis and find a more reasonable number (or prove that it actually does need to be 2500). A too large number will cause false positives once the set bonus comes into play.

Reply:
I did confirm via logs. I tried 2000 and there was stuff getting cut off from it
![image](https://user-images.githubusercontent.com/1304554/156889848-0b81f2b2-434b-4af4-8a42-343c70bea7b3.png)

Keep in mind that when you cast Comet Storm (Or Meteor for Fire), it doesnt do anything for like a second other than start the animation up in the sky ... and then it drops.

Reply:
Oh I knew there was a delay but I didn't realize it was that long - this is enough proof for me you can leave it as is